charts/datadog-operator/templates/_helpers.tpl

```diff
@@ -71,5 +71,20 @@ Return secret name to be used based on provided values.
 Return the appropriate apiVersion for PodDisruptionBudget policy APIs.
 */}}
 {{- define "policy.poddisruptionbudget.apiVersion" -}}
+{{- if or (.Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget") (semverCompare ">=1.21" .Capabilities.KubeVersion.Version) -}}
```

Comment:
Unless they changed something in their validation, `.Capabilities` check will be rejected. Anyway, this conditional can be dropped since EKS doesn't support pre-1.24 version.